src/coincident/io/download.py

```diff
@@ -266,15 +271,14 @@ def download_usgs_dem(
 
     # Use the helper function to download files.
     # the 'product' argument here is just for the TQDM progress bar statement
-    _download_files(
-        filtered_api_items, path=output_dir, product="DEM", url_key="downloadURL"
+    download_files(
+        filtered_api_items, output_dir=output_dir, product="DEM", url_key="downloadURL"
     )
 
 
-def fetch_usgs_lpc_tiles(
+def _fetch_usgs_lpc_tiles(
```

Comment:
Remember if changing function names to also update the API docs here (since we explicitly list all the methods rather than autogenerate them) https://github.com/uw-cryo/coincident/blob/2abafe5fb7c2d954ba8c488b524987ddcc526ca5/docs/api.rst?plain=1#L66-L69